gluoncv/data/mhp.py

```diff
@@ -0,0 +1,182 @@
+"""Multi-Human-Parsing V1 Dataset."""
+import os
+import numpy as np
+from PIL import Image
+from PIL import ImageFile
+
+import mxnet as mx
+from .segbase import SegmentationDataset
+
+ImageFile.LOAD_TRUNCATED_IMAGES = True
```

Comment:
Can you explain why use it and what is it? Thank you.

Reply:
It is used to handle the error (OSError: image file is truncated (2 bytes not processed)) while we reading images.
We see that PIL is reading in blocks of the file and that it expects that the blocks are going to be of a certain size. It turns out that we can ask PIL to be tolerant of files that are truncated (missing some file from the block) by changing a setting.